docs/source/contribution.rst

```diff
@@ -123,7 +123,42 @@ It is also appreciated to make such a backport PR by any contributors, though, s
 Issues and Pull Requests
 ------------------------
 
-In this section, we explain how to send pull requests (PRs).
+In this section, we explain how to send issues and pull requests (PRs).
+
+How to File an Issue
+~~~~~~~~~~~~~~~~~~~~
+
+(Work in progress.)
+
+AI Policies
+...........
+
+Recently, CuPy has received an enormous amount of bug reports in which AI/LLM tools were exploited to search
+for, e.g., unexpected corner cases of CuPy in which NumPy's behaviors are mismatched. We would like to reiterate
+that CuPy being a community project is managed and maintained by humans, not bots or AIs, and human beings
+have limited bandwidth and capacity in triaging and responding to bug reports in addition to other development
+activities. In light of recent incidents, we develop the following guidelines:
+
+- Please do not spam CuPy's issue tracker with AI-based contents.
+- Please do not misuse CuPy as a testbed for your AI development or experiment.
+- Please understand :doc:`certain deviations from NumPy <user_guide/difference>` are expected.
+- Please only file bug reports that actually impacts daily/production use of CuPy, not for hypothetical/pedantic
+  scenarios.
+- Please make sure bug reports are *not filed anonymously* (see below).
```

Comment:
I'm wondering if this is a bit aggressive ask to contributors. I guess policy items above are enough to cover cases that we'd want them to avoid?

Reply:
Let me add a bit of reasoning why this is discouraged.

Reply:
Addressed in commit fd280f26288d7e30ff7c7a9fe3cb413392b9d14d.

Reply:
As per the offline discussion, I've removed the anonymous profile part.